src/roman_datamodels/maker_utils/_ref_files.py

```diff
@@ -565,6 +567,169 @@ def mk_saturation(*, shape=(4096, 4096), filepath=None, **kwargs):
     return save_node(saturationref, filepath=filepath)
 
 
+def mk_roman_skycells(*, filepath=None, **kwargs):
+    skycellref = stnode.RomanSkycellsRef()
+    skycellref["meta"] = mk_ref_skycells_meta(**kwargs)
+    proj_dtype = np.dtype(
+        [
+            ("index", "<i4"),
+            ("ra_tangent", "<f8"),
+            ("dec_tangent", "<f8"),
+            ("ra_min", "<f8"),
+            ("ra_max", "<f8"),
+            ("dec_min", "<f8"),
+            ("dec_max", "<f8"),
+            ("orientat", "<f4"),
+            ("x_tangent", "<f8"),
+            ("y_tangent", "<f8"),
+            ("nx", "<i4"),
+            ("ny", "<i4"),
+            ("skycell_start", "<i4"),
+            ("skycell_end", "<i4"),
+            ("nxy_skycell", "<i4"),
+            ("skycell_border_pixels", "<i4"),
+            ("pixel_scale", "<f4"),
+        ]
+    )
+    skycell_dtype = np.dtype(
+        [
+            ("name", "<U16"),
+            ("ra_center", "<f8"),
+            ("dec_center", "<f8"),
+            ("orientat", "<f4"),
+            ("x_tangent", "<f8"),
+            ("y_tangent", "<f8"),
+            ("ra_corn1", "<f8"),
+            ("dec_corn1", "<f8"),
+            ("ra_corn2", "<f8"),
+            ("dec_corn2", "<f8"),
+            ("ra_corn3", "<f8"),
+            ("dec_corn3", "<f8"),
+            ("ra_corn4", "<f8"),
+            ("dec_corn4", "<f8"),
+        ]
+    )
+    proj_tab = np.array(
```

Comment:
Is it possible to fill these with empty arrays with the structured dtype rather than hard-coding data here?

Reply:
I think so, though there is a minor benefit for showing realistic values. I'm happy to change it though (after this whole name business is settled.